src/sentry/issues/occurrence_consumer.py

```diff
@@ -319,6 +320,21 @@ def process_occurrence_message(
         txn.set_tag("result", "dropped_feature_disabled")
         return None
 
+    # Rate limit based on fingerprint
+    rate_limit_key = f"occurrence_rate_limit:{occurrence_data['fingerprint']}"
+    if ratelimiter.backend.is_limited(
+        rate_limit_key,
+        limit=300,
```

Comment:
when we go to configure the rate limiter, we'll want the parameters to be options so we can fine tune / rapidly update them.